apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx

```diff
@@ -8,6 +8,84 @@ export const metadata = {
 
 # Migration Guide
 
+## v1.7
+
+Formbricks v1.7 comes with a big new feature like Multi-Language Surveys and other features to help you minimize the friction in reaching out to your users. This also involves a few changes in our environment variables. This guide will help you migrate your existing Formbricks instance to v1.7 without any hassles or build errors.
```

Comment:
`This also involves a few changes in our environment variables`
What are the changes in the env variables? Or is this copy/paste from the last release?

Reply:
yep i wrote that safer side beforehand if there were any, removed it for now